mllib/src/main/scala/org/apache/spark/mllib/regression/StreamingLinearAlgorithm.scala

```diff
@@ -114,7 +114,7 @@ abstract class StreamingLinearAlgorithm[
     if (model.isEmpty) {
       throw new IllegalArgumentException("Model must be initialized before starting prediction.")
     }
-    data.map(model.get.predict)
```

Comment:
Ah, I didn't realize that differed but it makes sense. So this line of change is actually the fix then? then I wasn't clear what shadowing `model` does in the subclass.